# Report for ForestQuery into Global Deforestation, 1990 to 2016

ForestQuery is on a mission to combat deforestation around the world and to raise awareness about this topic and its impact on the environment. The data analysis team at ForestQuery has obtained data from the World Bank that includes forest area and total land area by country and year from 1990 to 2016, as well as a table of countries and the regions to which they belong.

The data analysis team has used SQL to bring these tables together and to query them in an effort to find areas of concern as well as areas that present an opportunity to learn from successes.


In [1]:
print("Executing postgresql setup...")
%reload_ext sql
%sql postgresql://jadug.parusa:@localhost/deforestation
%sql SELECT * FROM forest_area WHERE forest_area_sqkm>0 ORDER BY 3 DESC, 4 DESC LIMIT 3

Executing postgresql setup...
 * postgresql://jadug.parusa:***@localhost/deforestation
3 rows affected.


[('WLD', 'World', 2016, Decimal('39958245.9')),
 ('RUS', 'Russian Federation', 2016, Decimal('8148895')),
 ('BRA', 'Brazil', 2016, Decimal('4925540'))]

In [2]:
%%sql

-- #DROP VIEW IF EXISTS deforestation_tab;
CREATE OR REPLACE VIEW deforestation_tab AS (
    SELECT
        f.country_code,
        f.country_name,
        f.year,
        f.forest_area_sqkm,
        (2.59 * l.total_area_sqmi) AS total_area_sqkm,
        r.region,
        r.income_group
    FROM forest_area AS f
    FULL OUTER JOIN
        land_area AS l
        ON
            f.country_code = l.country_code
        AND
            f.year = l.year
    FULL OUTER JOIN
        regions AS r
        ON
            f.country_code = r.country_code
);
SELECT *, COUNT(*) over()
FROM deforestation_tab
WHERE forest_area_sqkm>0
GROUP BY 1,2,3,4,5,6,7
ORDER BY 3 DESC, 4 DESC
LIMIT 3;

 * postgresql://jadug.parusa:***@localhost/deforestation
Done.
3 rows affected.


[('WLD', 'World', 2016, Decimal('39958245.9'), Decimal('127354641.4357'), 'World', 'NULL', 5570),
 ('RUS', 'Russian Federation', 2016, Decimal('8148895'), Decimal('16376869.9997'), 'Europe & Central Asia', 'Upper middle income', 5570),
 ('BRA', 'Brazil', 2016, Decimal('4925540'), Decimal('8358139.9972'), 'Latin America & Caribbean', 'Upper middle income', 5570)]

## 1. GLOBAL SITUATION

#### Question 1.1: How was the World's 1990 total forest area compared to 2016?

In [3]:
%%sql
-- # filter country (world), year (1990 or 2016), calculate (area, difference, and percentage)
DROP VIEW IF EXISTS world_forest_loss;
CREATE VIEW world_forest_loss AS (
    WITH
    world_1990_2016 AS (
        SELECT
            country_name,
            year,
            forest_area_sqkm
        FROM
            deforestation_tab
        WHERE
            country_name = 'World'
            AND (
                year = 1990
                OR year = 2016
            )
    ),
    forest_1990 AS (
        SELECT
            forest_area_sqkm AS base_1990_sqkm
        FROM
            world_1990_2016
        WHERE
            year = 1990
    )

    SELECT
        t.country_name,
        t.year,
        t.forest_area_sqkm,
        base_1990_sqkm,
        (t.forest_area_sqkm - base_1990_sqkm) AS forest_change_sqkm,
        100*ROUND((t.forest_area_sqkm - base_1990_sqkm) / base_1990_sqkm, 3) AS forest_change_percent
    FROM
        world_1990_2016 AS t,
        forest_1990
    ORDER BY 2
);

SELECT * FROM world_forest_loss

 * postgresql://jadug.parusa:***@localhost/deforestation
Done.
Done.
2 rows affected.


[('World', 1990, Decimal('41282694.9'), Decimal('41282694.9'), Decimal('0.0'), Decimal('0.000')),
 ('World', 2016, Decimal('39958245.9'), Decimal('41282694.9'), Decimal('-1324449.0'), Decimal('-3.200'))]

#### Question 1.2: How comparable is this forest area loss against a single country size?

In [4]:
%%sql
-- # filter year (2016), filter and order (by total_area_sqkm)
WITH base_2016_loss AS (
    SELECT
        forest_change_sqkm AS world_loss_sqkm
    FROM
        world_forest_loss
    WHERE
        year = 2016
)

SELECT
    country_name,
    year,
    region,
    total_area_sqkm
FROM
    deforestation_tab AS d,
    base_2016_loss
WHERE
    year = 2016
    AND d.total_area_sqkm < -(world_loss_sqkm)
ORDER BY 4 DESC
LIMIT 3

 * postgresql://jadug.parusa:***@localhost/deforestation
3 rows affected.


[('Peru', 2016, 'Latin America & Caribbean', Decimal('1279999.9891')),
 ('Niger', 2016, 'Sub-Saharan Africa', Decimal('1266700.0024')),
 ('Chad', 2016, 'Sub-Saharan Africa', Decimal('1259200.0099'))]

According to the World Bank, the total forest area of the world was `41,282,694.9 sqkm` in 1990. As of 2016, the most recent year for which data was available, that number had fallen to `39,958,245.9 sqkm`, a loss of `-1,324,449.0 sqkm`, or `-3.200%`.

The forest area lost over this time period is slightly more than the entire land area of `Peru` listed for the year 2016 (which is `1,279,999.9891 sqkm`).


## 2. REGIONAL OUTLOOK

#### Question 2.1:
#### How many percent of World's land area was designated as forest, in 2016 and 1990?
#### Which region had the highest & the lowest forestation percentage respectively in 2016 and 1990?

In [5]:
%%sql
-- # filter year (1990 or 2016), calculate (forest area, land area, and percentage),
-- # select world's, highest region, lowest region
-- # apply union
DROP VIEW IF EXISTS region_forestation;
CREATE VIEW region_forestation AS (
    SELECT
        region,
        year,
        ROUND(100 * SUM(forest_area_sqkm) / SUM(total_area_sqkm), 3) AS forestation_percent,
        ROUND(SUM(forest_area_sqkm),3) AS region_forest_sqkm,
        ROUND(SUM(total_area_sqkm),3) AS region_land_sqkm
    FROM
        deforestation_tab
    WHERE
        year = 2016
        OR year = 1990
        AND COALESCE(forest_area_sqkm,0) > 0 -- excluding countries having empty forest_area_sqkm
        AND COALESCE(total_area_sqkm,0) > 0 -- excluding countries having empty total_area_sqkm
    GROUP BY 1,2
    ORDER BY 2, 3 DESC
);
WITH
world_forestation AS (
    SELECT * FROM region_forestation WHERE region = 'World' LIMIT 2
),
highest_2016 AS (
    SELECT * FROM region_forestation WHERE year = 2016 ORDER BY 3 DESC LIMIT 1
),
lowest_2016 AS (
    SELECT * FROM region_forestation WHERE year = 2016 ORDER BY 3 LIMIT 1
),
highest_1990 AS (
    SELECT * FROM region_forestation WHERE year = 1990 ORDER BY 3 DESC LIMIT 1
),
lowest_1990 AS (
    SELECT * FROM region_forestation WHERE year = 1990 ORDER BY 3 LIMIT 1
)

SELECT * FROM world_forestation
UNION
SELECT * FROM highest_2016
UNION
SELECT * FROM lowest_2016
UNION
SELECT * FROM highest_1990
UNION
SELECT * FROM lowest_1990

ORDER BY 1 DESC,2

 * postgresql://jadug.parusa:***@localhost/deforestation
Done.
Done.
6 rows affected.


[('World', 1990, Decimal('32.422'), Decimal('41282694.900'), Decimal('127328467.440')),
 ('World', 2016, Decimal('31.376'), Decimal('39958245.900'), Decimal('127354641.436')),
 ('Middle East & North Africa', 1990, Decimal('1.777'), Decimal('199292.596'), Decimal('11214619.995')),
 ('Middle East & North Africa', 2016, Decimal('2.068'), Decimal('232131.004'), Decimal('11223465.985')),
 ('Latin America & Caribbean', 1990, Decimal('51.031'), Decimal('10242341.796'), Decimal('20070746.441')),
 ('Latin America & Caribbean', 2016, Decimal('46.162'), Decimal('9250585.884'), Decimal('20039364.447'))]

In 2016, the percent of the total land area of the world designated as forest was `31.376%`. The region with the highest relative forestation was `Latin America & Caribbean`, with `46.162%`, and the region with the lowest relative forestation was `Middle East & North Africa`, with `2.068%` forestation.

In 1990, the percent of the total land area of the world designated as forest was `32.422%`. The region with the highest relative forestation was `Latin America & Caribbean`, with `51.031%`, and the region with the lowest relative forestation was `Middle East & North Africa`, with `1.777%` forestation.


#### Question 2.2:
#### From 1990 to 2016, which regions had increasing forestation & which regions had decreasing forestation?
#### How did those regional increase & decrease impact to the overall World' forestation?

In [6]:
%%sql
-- # self join to combine percentage (1990 or 2016)
WITH
rf_selfjoin AS (
    SELECT
        rf_1990.region,
        rf_1990.forestation_percent AS forestation_1990_percent,
        rf_2016.forestation_percent AS forestation_2016_percent
    FROM region_forestation AS rf_1990
    JOIN region_forestation AS rf_2016
    ON
        rf_1990.region = rf_2016.region
        AND rf_1990.year = 1990
        AND rf_2016.year = 2016
)

SELECT
    region,
    forestation_1990_percent,
    forestation_2016_percent,
    ROUND(100*(forestation_2016_percent - forestation_1990_percent)/forestation_1990_percent, 3) AS percent_change
FROM
    rf_selfjoin
ORDER BY 4

 * postgresql://jadug.parusa:***@localhost/deforestation
8 rows affected.


[('Sub-Saharan Africa', Decimal('32.190'), Decimal('28.788'), Decimal('-10.568')),
 ('Latin America & Caribbean', Decimal('51.031'), Decimal('46.162'), Decimal('-9.541')),
 ('World', Decimal('32.422'), Decimal('31.376'), Decimal('-3.226')),
 ('North America', Decimal('35.651'), Decimal('36.039'), Decimal('1.088')),
 ('Europe & Central Asia', Decimal('37.271'), Decimal('38.041'), Decimal('2.066')),
 ('East Asia & Pacific', Decimal('25.771'), Decimal('26.359'), Decimal('2.282')),
 ('South Asia', Decimal('16.511'), Decimal('17.506'), Decimal('6.026')),
 ('Middle East & North Africa', Decimal('1.777'), Decimal('2.068'), Decimal('16.376'))]

The only regions of the world that decreased in percent forest area from 1990 to 2016 were `Latin America & Caribbean` (dropped from `51.031%` to `46.162%`) and `Sub-Saharan Africa` (`32.190%` to `28.788%`).

All other regions actually increased in forest area over this time period. However, the drop in forest area in the two aforementioned regions was so large, the percent forest area of the world decreased over this time period from `32.422%` to `31.376%`. 


## 3. COUNTRY-LEVEL DETAIL

### A. SUCCESS STORIES

#### Question 3.1:
#### Which top 2 countries had the largest increase of forest area from 1990 to 2016?

In [7]:
%%sql
-- # filter year (1990 or 2016), calculate (forest area, land area, and percentage),
-- # self join to combine percentage (1990 or 2016)
-- # select highest countries

DROP VIEW IF EXISTS country_forest_area;
CREATE VIEW country_forest_area AS (
    SELECT
        d_1990.country_name,
        d_1990.region,
        ROUND(SUM(d_1990.total_area_sqkm), 3) AS land_1990_sqkm,
        ROUND(SUM(d_2016.total_area_sqkm), 3) AS land_2016_sqkm,
        ROUND(SUM(d_1990.forest_area_sqkm), 3) AS forest_1990_sqkm,
        ROUND(SUM(d_2016.forest_area_sqkm), 3) AS forest_2016_sqkm,
        ROUND(100 * SUM(d_1990.forest_area_sqkm) / SUM(d_1990.total_area_sqkm), 3) AS forestation_1990_percent,
        ROUND(100 * SUM(d_2016.forest_area_sqkm) / SUM(d_2016.total_area_sqkm), 3) AS forestation_2016_percent
    FROM
        deforestation_tab AS d_1990
    JOIN
        deforestation_tab AS d_2016
    ON
        d_1990.country_name = d_2016.country_name
        AND d_1990.year = 1990
        AND d_2016.year = 2016
    WHERE
        COALESCE(d_1990.total_area_sqkm, 0) > 0 -- excluding countries having empty land_area_sqkm
        AND COALESCE(d_2016.total_area_sqkm, 0) > 0 -- excluding countries having empty land_area_sqkm
        AND COALESCE(d_1990.forest_area_sqkm, 0) > 0 -- excluding countries having empty forest_area_sqkm
        AND COALESCE(d_2016.forest_area_sqkm, 0) > 0 -- excluding countries having empty forest_area_sqkm
    GROUP BY 1,2
);

SELECT
    country_name,
    region,
    land_2016_sqkm,
    forest_1990_sqkm,
    forest_2016_sqkm,
    (forest_2016_sqkm - forest_1990_sqkm) AS forest_sqkm_change
FROM
    country_forest_area
ORDER BY
    forest_sqkm_change DESC
LIMIT 5

 * postgresql://jadug.parusa:***@localhost/deforestation
Done.
Done.
5 rows affected.


[('China', 'East Asia & Pacific', Decimal('9388210.011'), Decimal('1571405.938'), Decimal('2098635.000'), Decimal('527229.062')),
 ('United States', 'North America', Decimal('9147419.990'), Decimal('3024500.000'), Decimal('3103700.000'), Decimal('79200.000')),
 ('India', 'South Asia', Decimal('2973190.008'), Decimal('639390.000'), Decimal('708603.984'), Decimal('69213.984')),
 ('Russian Federation', 'Europe & Central Asia', Decimal('16376870.000'), Decimal('8089500.000'), Decimal('8148895.000'), Decimal('59395.000')),
 ('Vietnam', 'East Asia & Pacific', Decimal('310070.009'), Decimal('93630.000'), Decimal('149020.000'), Decimal('55390.000'))]

There is one particularly bright spot in the data at the country level, `China`. This country actually increased in forest area from 1990 to 2016 by `527,229.062 sqkm`. It would be interesting to study what has changed in this country over this time to drive this figure in the data higher.

The country with the next largest increase in forest area from 1990 to 2016 was the `United States`, but it only saw an increase of `79,200.000 sqkm`, much lower than the figure for `China`.


#### Question 3.2:
#### Which top 2 countries had the largest land area in 2016?

In [8]:
%%sql

SELECT
    country_name,
    region,
    land_2016_sqkm,
    (forest_2016_sqkm - forest_1990_sqkm) AS forest_sqkm_change,
    forestation_1990_percent,
    forestation_2016_percent,
    ROUND(100*(forestation_2016_percent - forestation_1990_percent)/forestation_1990_percent, 3) AS percent_change
FROM
    country_forest_area
WHERE
    country_name != 'World'
ORDER BY
    land_2016_sqkm DESC
LIMIT 5

 * postgresql://jadug.parusa:***@localhost/deforestation
5 rows affected.


[('Russian Federation', 'Europe & Central Asia', Decimal('16376870.000'), Decimal('59395.000'), Decimal('49.356'), Decimal('49.759'), Decimal('0.817')),
 ('China', 'East Asia & Pacific', Decimal('9388210.011'), Decimal('527229.062'), Decimal('16.738'), Decimal('22.354'), Decimal('33.552')),
 ('United States', 'North America', Decimal('9147419.990'), Decimal('79200.000'), Decimal('33.022'), Decimal('33.930'), Decimal('2.750')),
 ('Canada', 'North America', Decimal('9093509.995'), Decimal('-12505.937'), Decimal('38.299'), Decimal('38.162'), Decimal('-0.358')),
 ('Brazil', 'Latin America & Caribbean', Decimal('8358139.997'), Decimal('-541510.000'), Decimal('65.410'), Decimal('58.931'), Decimal('-9.905'))]

#### Question 3.3:
#### Which top 5 countries had the largest increase of forestation percentage from 1990 to 2016?

In [9]:
%%sql

SELECT
    country_name,
    region,
    land_2016_sqkm,
    (forest_2016_sqkm - forest_1990_sqkm) AS forest_sqkm_change,
    forestation_1990_percent,
    forestation_2016_percent,
    ROUND(100*(forestation_2016_percent - forestation_1990_percent)/forestation_1990_percent, 3) AS percent_change
FROM
    country_forest_area
ORDER BY
    percent_change DESC
LIMIT 5

 * postgresql://jadug.parusa:***@localhost/deforestation
5 rows affected.


[('Iceland', 'Europe & Central Asia', Decimal('100249.990'), Decimal('344.000'), Decimal('0.161'), Decimal('0.504'), Decimal('213.043')),
 ('French Polynesia', 'East Asia & Pacific', Decimal('3660.007'), Decimal('1000.000'), Decimal('15.027'), Decimal('42.350'), Decimal('181.826')),
 ('Bahrain', 'Middle East & North Africa', Decimal('778.010'), Decimal('3.900'), Decimal('0.319'), Decimal('0.784'), Decimal('145.768')),
 ('Uruguay', 'Latin America & Caribbean', Decimal('175020.001'), Decimal('10699.400'), Decimal('4.558'), Decimal('10.672'), Decimal('134.138')),
 ('Dominican Republic', 'Latin America & Caribbean', Decimal('48310.001'), Decimal('9112.000'), Decimal('22.873'), Decimal('41.735'), Decimal('82.464'))]

`Russian Federation` (`16,376,870.000 sqkm`) and `China` (`9,388,210.011 sqkm`) are of course very large countries in total land area, so when we look at the largest percent change in forest area from 1990 to 2016, we aren’t surprised to find a much smaller country listed at the top.

`Iceland` (`100,249.990 sqkm`) increased in forest area by `213.043%` from 1990 to 2016. 


### B. LARGEST CONCERNS

#### Question 3.4: Which countries are seeing deforestation to the largest degree from 1990 to 2016? (by sqkm decrease in forest area)

The following 3 countries had the largest decrease in forest area over the time period under consideration:

In [10]:
%%sql

SELECT
    country_name,
    region,
    land_2016_sqkm,
    forest_1990_sqkm,
    forest_2016_sqkm,
    (forest_2016_sqkm - forest_1990_sqkm) AS forest_sqkm_change
FROM
    country_forest_area
WHERE
    country_name != 'World'
ORDER BY
    forest_sqkm_change
LIMIT 5

 * postgresql://jadug.parusa:***@localhost/deforestation
5 rows affected.


[('Brazil', 'Latin America & Caribbean', Decimal('8358139.997'), Decimal('5467050.000'), Decimal('4925540.000'), Decimal('-541510.000')),
 ('Indonesia', 'East Asia & Pacific', Decimal('1811570.009'), Decimal('1185450.000'), Decimal('903256.016'), Decimal('-282193.984')),
 ('Myanmar', 'East Asia & Pacific', Decimal('653080.000'), Decimal('392180.000'), Decimal('284945.996'), Decimal('-107234.004')),
 ('Nigeria', 'Sub-Saharan Africa', Decimal('910770.004'), Decimal('172340.000'), Decimal('65833.999'), Decimal('-106506.001')),
 ('Tanzania', 'Sub-Saharan Africa', Decimal('885799.995'), Decimal('559200.000'), Decimal('456880.000'), Decimal('-102320.000'))]

#### Question 3.5: Which countries are seeing deforestation to the largest degree from 1990 to 2016? (by percent decrease)

In [11]:
%%sql

SELECT
    country_name,
    region,
    land_2016_sqkm,
    (forest_2016_sqkm - forest_1990_sqkm) AS forest_sqkm_change,
    forestation_1990_percent,
    forestation_2016_percent,
    ROUND(100*(forestation_2016_percent - forestation_1990_percent)/forestation_1990_percent, 3) AS percent_change
FROM
    country_forest_area
ORDER BY
    percent_change
LIMIT 5

 * postgresql://jadug.parusa:***@localhost/deforestation
5 rows affected.


[('Togo', 'Sub-Saharan Africa', Decimal('54390.000'), Decimal('-5168.000'), Decimal('12.594'), Decimal('3.092'), Decimal('-75.449')),
 ('Nigeria', 'Sub-Saharan Africa', Decimal('910770.004'), Decimal('-106506.001'), Decimal('18.922'), Decimal('7.228'), Decimal('-61.801')),
 ('Uganda', 'Sub-Saharan Africa', Decimal('200520.002'), Decimal('-28092.000'), Decimal('23.778'), Decimal('9.684'), Decimal('-59.273')),
 ('Mauritania', 'Sub-Saharan Africa', Decimal('1030700.005'), Decimal('-1940.000'), Decimal('0.403'), Decimal('0.214'), Decimal('-46.898')),
 ('Honduras', 'Latin America & Caribbean', Decimal('111889.994'), Decimal('-36640.000'), Decimal('72.714'), Decimal('39.968'), Decimal('-45.034'))]

When we consider countries that decreased in forest area percentage the most between 1990 and 2016, we find that four of the top 5 countries on the list are in the region of `Sub-Saharan Africa`. The countries are `Togo`, `Nigeria`, `Uganda`, and `Mauritania`. The 5th country on the list is `Honduras`, which is in the `Latin America & Caribbean` region. 

From the above analysis, we see that `Nigeria` is the only country that ranks in the top 5 both in terms of absolute square kilometer decrease in forest as well as percent decrease in forest area from 1990 to 2016. Therefore, this country has a significant opportunity ahead to stop the decline and hopefully spearhead remedial efforts.


### C. QUARTILES

In [12]:
%%sql

DROP VIEW IF EXISTS country_quartile;
CREATE VIEW country_quartile AS (
    WITH country_2016_forestation_percent AS (
        SELECT
            country_name,
            region,
            forest_area_sqkm,
            total_area_sqkm,
            ROUND(100 * forest_area_sqkm / total_area_sqkm, 3) AS forestation_percent
        FROM
            deforestation_tab
        WHERE
            year = 2016
            AND COALESCE(total_area_sqkm, 0) > 0 -- excluding countries having empty land_area_sqkm
            AND COALESCE(forest_area_sqkm, 0) > 0 -- excluding countries having empty forest_area_sqkm
        GROUP BY 1,2,3,4
        ORDER BY 5 DESC
    )

    SELECT
        country_name,
        region,
        forestation_percent,
        CASE
            WHEN forestation_percent BETWEEN 0 AND 25 THEN '1st Quartile (0-25%)'
            WHEN forestation_percent BETWEEN 25 AND 50 THEN '2nd Quartile (25-50%)'
            WHEN forestation_percent BETWEEN 50 AND 75 THEN '3rd Quartile (50-75%)'
            WHEN forestation_percent BETWEEN 75 AND 100 THEN '4th Quartile (75-100%)'
        END AS quartile
    FROM
        country_2016_forestation_percent
    ORDER BY 1
);


SELECT
    quartile,
    count(country_name)
FROM 
    country_quartile
GROUP BY 1
ORDER BY 1

 * postgresql://jadug.parusa:***@localhost/deforestation
Done.
Done.
4 rows affected.


[('1st Quartile (0-25%)', 85),
 ('2nd Quartile (25-50%)', 73),
 ('3rd Quartile (50-75%)', 38),
 ('4th Quartile (75-100%)', 9)]

The largest number of countries in 2016 were found in the `1st quartile`.

There were `9` countries in the top quartile in 2016. These are countries with a very high percentage of their land area designated as forest. The following is a list of countries and their respective forest land, denoted as a percentage.


In [13]:
%%sql

SELECT
    country_name,
    region,
    forestation_percent
FROM 
    country_quartile
WHERE
    quartile = '4th Quartile (75-100%)'
ORDER BY 3 DESC

 * postgresql://jadug.parusa:***@localhost/deforestation
9 rows affected.


[('Suriname', 'Latin America & Caribbean', Decimal('98.258')),
 ('Micronesia, Fed. Sts.', 'East Asia & Pacific', Decimal('91.857')),
 ('Gabon', 'Sub-Saharan Africa', Decimal('90.038')),
 ('Seychelles', 'Sub-Saharan Africa', Decimal('88.411')),
 ('Palau', 'East Asia & Pacific', Decimal('87.607')),
 ('American Samoa', 'East Asia & Pacific', Decimal('87.500')),
 ('Guyana', 'Latin America & Caribbean', Decimal('83.901')),
 ('Lao PDR', 'East Asia & Pacific', Decimal('82.108')),
 ('Solomon Islands', 'East Asia & Pacific', Decimal('77.864'))]

## 4. RECOMMENDATIONS

### Group 1. Countries with SUCCESS STORIES: Find out insights and learn from them!

These countries are the role model! It would bring huge environmental benefits to study what has changed in this country over this time to drive higher forestation, and proliferate into other country groups below.

### Group 2. Countries with LARGEST CONCERNS: Prioritize improvement!

These countries need help and attention. It would be necessary to find out what has caused the issues and how to cure the situation. Learn from Group 1!

### Group 3. Countries in top forestation quartile: Preserve carefully.

These countries must be protected with high carefulness. It is necessary to prevent them from falling into Group 2.

### Group 4: Countries in bottom forestation quartile: Maintain and seek opportunity.

There is nothing much to be done if these countries are located where climate does not really support, except maintaining normal level of forestation. However if the climate supports, learn from Group 1.